tests/TestCase/Error/ExceptionRendererTest.php

```diff
@@ -583,21 +584,21 @@ public static function exceptionProvider()
                 500
             ],
             [
-                new \Exception('boom'),
+                new Exception('boom'),
                 [
                     '/Internal Error/'
                 ],
                 500
             ],
             [
-                new \RuntimeException('another boom'),
+                new RuntimeException('another boom'),
                 [
                     '/Internal Error/'
                 ],
                 500
             ],
             [
-                new \Cake\Core\Exception\Exception('base class'),
+                new CakeException('base class'),
```

Comment:
Hi,

The PHP Exception is being used in this file also so to avoid confusion I renamed the Exception inside Cake namespace to have the CakeException alias to signify this is a Cake exception class

> On 6 Apr 2015, at 00:54, Jad Bitar <notifications@github.com> wrote:
> 
> In tests/TestCase/Error/ExceptionRendererTest.php:
> 
> >                  [
> >                      '/Internal Error/'
> >                  ],
> >                  500
> >              ],
> >              [
> > -                new \Cake\Core\Exception\Exception('base class'),
> > +                new CakeException('base class'),
> Why was this changed to CakeException?
> 
> —
> Reply to this email directly or view it on GitHub.
>